roles/lib_openshift/src/lib/base.py

```diff
@@ -243,7 +266,10 @@ def openshift_cmd(self, cmd, oadm=False, output=False, output_type='json', input
         if self.verbose:
             print(' '.join(cmds))
 
-        returncode, stdout, stderr = self._run(cmds, input_data)
+        try:
+            returncode, stdout, stderr = self._run(cmds, input_data)
```

Comment:
I think we can live without a dependency on `sh`. It's not our core business to call external commands. I'd stick with the stdlib.